src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/SamlAssertionKeyIdentifierClause.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CoreWCF.IdentityModel
+{
+    using System.Globalization;
+
+    public class SamlAssertionKeyIdentifierClause : SecurityKeyIdentifierClause
+    {
```

Comment:
We don't need this yet. And when we do need this, it will be in a separate package. This is for enabling IssuedToken support when using WSTrust